<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Music</class>
 <widget class="QMainWindow" name="Music">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">Music{border-image: url(:/mic.bmp)};</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/add.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>180</width>
      <height>330</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>10</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/main.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_prev">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/music/prev.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_play">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>390</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/music/pause.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_next">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>390</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/music/next.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>390</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/music/volume.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>400</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00/00:00</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>400</y>
      <width>431</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal { 
border: 1px solid #55ffff; 
background: white; 
height: 10px; 
border-radius: 4px; 
} 
   
QSlider::sub-page:horizontal { 
background: qlineargradient(x1: 0, y1: 0.2, x2: 1, y2: 1, 
    stop: 0 #55ffff, stop: 1 #55ffff); 
border: 1px solid #777; 
height: 10px; 
border-radius: 4px; 
} 
   
QSlider::add-page:horizontal { 
background: #fff; 
border: 1px solid #777; 
height: 10px; 
border-radius: 4px; 
} 
   
QSlider::handle:horizontal { 
background: qlineargradient(x1:0, y1:0, x2:1, y2:1, 
    stop:0 #eee, stop:1 #ccc); 
border: 1px solid #777; 
width: 13px; 
margin-top: -2px; 
margin-bottom: -2px; 
border-radius: 4px; 
} 
   
QSlider::handle:horizontal:hover { 
background: qlineargradient(x1:0, y1:0, x2:1, y2:1, 
    stop:0 #fff, stop:1 #ddd); 
border: 1px solid #444; 
border-radius: 4px; 
} 
   
QSlider::sub-page:horizontal:disabled { 
background: #bbb; 
border-color: #999; 
} 
   
QSlider::add-page:horizontal:disabled { 
background: #eee; 
border-color: #999; 
} 
   
QSlider::handle:horizontal:disabled { 
background: #eee; 
border: 1px solid #aaa; 
border-radius: 4px; 
} </string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>230</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider
{
    background-color: rgba(22, 22, 22, 0.7);
    padding-top: 15px;  /*上面端点离顶部的距离*/
    padding-bottom: 15px;
    border-radius: 5px; /*外边框矩形倒角*/
}
  
QSlider::add-page:vertical
{
    background-color: #55ffff;
    width:5px;
    border-radius: 2px;
}
  
QSlider::sub-page:vertical
{
    background-color: #7A7B79;
    width:5px;
    border-radius: 2px;
}
  
QSlider::groove:vertical
{
    background:transparent;
    width:6px;
}
  
QSlider::handle:vertical   
{
    height: 14px; 
    width: 14px;
    margin: 0px -4px 0px -4px;
    border-radius: 7px;
    background: white;
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="del">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/del.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>501</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_word">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_word_2">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_word_3">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_word_4">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_word_5">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_word_6">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
